tests/integration/test_prometheus_integration.py

```diff
@@ -39,14 +41,31 @@ async def labels(self) -> List[str]:
         return result["data"] if result["status"] == "success" else []
 
 
+@pytest.fixture
+def updated_charmlib():
```

Comment:
Would be handy if this was an [`autouse` fixture](https://github.com/canonical/grafana-k8s-operator/blob/0a4430d2ddbe448a7933d64b645174889ba0b6e0/tests/integration/conftest.py#L55) - won't need to manually list it in test func args.

Reply:
I don't expect to *all* the integration tests to use the library. We can leave it like this for now and generalize in the future if needed.